earth_enterprise/src/common/packet.cpp

```diff
@@ -296,7 +296,7 @@ int etDataPacket::getFlatFileCount()
   char fname[500];
   int count;
 
-  sprintf(fname, "%s.header", flatbasename);
+  snprintf(fname, 500, "%s.header", flatbasename);
```

Comment:
probably should be sizeof(fname) in case anyone ever changes it, or have a size_t bufsize = 500, and use that for the array size and the snprintf.